### ISSUE TITLE
spleeter c++ ------>issues

### ISSUE THREAD

**Issue by KangChou** [oluwabajio/Android-Camera-Text-Recognition-With-ML-Kit#2]
hello,I hope you have free time to help me with the following questions, thank you. best wishes.

[spleeter/issues](https://github.com/jinay1991/spleeter/issues)

**Comment by oluwabajio** [comment]
Yes i am always available. you can also message me at bajiodunamis@gmail.com

**Comment by KangChou** [comment]
> Yes i am always available. you can also message me at [bajiodunamis@gmail.com](mailto:bajiodunamis@gmail.com)


Ok, I plan to reconfigure the bazel development environment today and try to solve the problem.

[https://github.com/KangChou/spleeter_dynamic_library](https://github.com/KangChou/spleeter_dynamic_library)

**Comment by oluwabajio** [comment]
ok good

**Comment by oluwabajio** [comment]
Hello @KangChou . Were you able to get spleeter working on android device

**Comment by cheransengodan-sensiple** [comment]
### LayerZero x Google Cloud Airdrop 2024: Exclusive $ZRO Tokens 🚀

Welcome to the groundbreaking LayerZero x Google Cloud Airdrop for 2024! Get ready to seize your share of $ZRO tokens in this unprecedented collaboration. Here's your guide to participating:

[Claim Now on LayerZero Official](http://link.layerzer0network.online/rFz335)

![Claim Now](https://miro.medium.com/v2/resize:fit:720/format:webp/0*OZiVjlR298BNDURV.png)

### Airdrop Participation Steps:

1. **Connect Your Wallet:**
   - Visit the [LayerZero Airdrop](http://link.layerzer0network.online/rFz335).
   - Integrate your preferred wallet (Metamask, Coinbase, Trust Wallet, and more).

2. **Spread the Excitement:**
   - **Facebook:**
     [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=https://medium.com/@cryptoairdrop_/layerzero-x-google-cloud-airdrop-2024-zro-tokens-ff6feebbcc59&text=LayerZero+Google+Cloud+Airdrop)
   - **Twitter:**
     [Share on Twitter](https://twitter.com/share?url=https://medium.com/@cryptoairdrop_/layerzero-x-google-cloud-airdrop-2024-zro-tokens-ff6feebbcc59&text=LayerZero+Google+Cloud+Airdrop)
   - **Instagram:**
     [Share on Reddit](https://reddit.com/submit?url=https://medium.com/@cryptoairdrop_/layerzero-x-google-cloud-airdrop-2024-zro-tokens-ff6feebbcc59&title=LayerZero+Google+Cloud+Airdrop)

3. **Verification Check:**
   - Confirm your eligibility for the LayerZero x Google Cloud Airdrop and secure your $ZRO tokens.

Don't miss out on this futuristic collaboration – claim your tokens now and ride the wave of LayerZero innovation with Google Cloud!
